rasa/nlu/components.py

```diff
@@ -51,17 +51,46 @@ def validate_requirements(component_names: List[Text]) -> None:
         )
 
 
+def validate_tokenizers(pipeline: List["Component"]) -> None:
+    """Validates that only one tokenizer is present in the pipeline."""
+
+    from rasa.nlu.tokenizers.tokenizer import Tokenizer
+
+    tokenizer_names = []
+    for component in pipeline:
+        if isinstance(component, Tokenizer):
+            tokenizer_names.append(component.name)
+
+    if len(tokenizer_names) > 1:
+        raise InvalidConfigError(
+            f"More then one tokenizer is used: {tokenizer_names}. "
+            f"You can use only one tokenizer."
+        )
+
+
+def validate_required_components(pipeline: List["Component"]) -> None:
```

Comment:
did this in the other pr, but should probably go here:
why do we need this? should this not be covered by a components `requires` property

Reply:
Partly, some components, such as `ConveRTFeaturizer`, require `tokens` from the `ConveRTTokenizer`. Just saying the `ConveRTFeaturizer` requires `tokens` would not be enough.